src/plugins/console/public/application/hooks/use_send_current_request_to_es/send_request_to_es.ts

```diff
@@ -88,7 +88,13 @@ export function sendRequestToES(args: EsRequestArgs): Promise<ESRequestResult[]>
             (response.status >= 200 && response.status < 300) || response.status === 404;
 
           if (isSuccess) {
-            let value = JSON.stringify(body, null, 2);
+            let value;
+            // check if object is ArrayBuffer
+            if ((body as ArrayBuffer).byteLength > 0) {
```

Comment:
Is there any reason we can't use?

```suggestion
            if (body instanceof ArrayBuffer) {
```